docs/multicast-guide.md

```diff
@@ -0,0 +1,122 @@
+# Multicast User Guide
+
+Antrea supports Multicast traffic in the following scenarios:
+
+1. Pods to Pods, any Pod that has joined the multicast group should receive the traffic from the Pod senders send multicast traffic to that group.
+2. Pods to External, external hosts connected to the Node will receive the multicast traffic from that local Pods.
```

Comment:
external hosts can receive the multicast traffic sent from Pods, when the Node network supports multicast forwarding / routing to the external hosts.

Reply:
updated